pandas/tests/apply/test_str.py

```diff
@@ -163,10 +166,10 @@ def test_agg_cython_table_transform_series(request, series, func, expected):
     # GH21224
     # test transforming functions in
     # pandas.core.base.SelectionMixin._cython_table (cumprod, cumsum)
-    if series.dtype == "string" and func == "cumsum":
+    if series.dtype == "string" and func == "cumsum" and not HAS_PYARROW:
```

Comment:
Can tackle the `StringArray` case in a followup or do so here if preferred.

Reply:
As you prefer!